content/en/admin/optional/captcha.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Captcha
+description: Mitigating automated signup bots
+menu:
+  docs:
+    weight: 30
+    parent: admin-optional
+---
+
+As of Mastodon 4.2, using CAPTCHA technology is supported to help mitigate against bots signing up for new accounts.
+With CAPTCHA enabled, new registrations will be required to complete a challenge response as part of the e-mail verification process.
+
+![](/assets/captcha/user-view.png)
+
+{{< hint style="danger" >}}
+For some people, the use of a central CAPTCHA service may be a security and privacy concern.
+In addition, CAPTCHA can make the registration process significantly less accessible to some (especially disabled) people.
```

Comment:
I would remove the bracketed section here -> "... less accessible to some people." is all that is required to make the point.

Reply:
Done. That line came from the interface itself, but I didn't like it much anyway.